CHANGELOG.md

```diff
@@ -1,665 +1,699 @@
 # 5.8.3 (in progress)
 
-* Your contribution here
+-   Your contribution here
 
 # 5.8.0 (2021-07-18), 5.8.1 (2021-08-22), 5.8.2 (2021-09-05)
 
 ##### New Features
-* [#1654](https://github.com/oshi/oshi/pull/1654): API for process arguments and environment - [@basil](https://github.com/basil) and [@dbwiddis](https://github.com/dbwiddis).
```

Comment:
Don't change the format of the entire file.  Is there a reason the existing format doesn't work well?

I don't see your addition here.

Please rebase to the latest version on master and add your entry to it.